src/alg/ccf/convenience.jl

```diff
@@ -148,10 +155,16 @@ function calc_ccf_chunklist_timeseries_expr(clt::AbstractChunkListTimeseries,
   num_lines = length(plan.line_list.λ)
   plan_for_chunk = Vector{BasicCCFPlan}(undef,num_chunks(clt))
   for chid in 1:num_chunks(clt)
+      # find the maximum lower wavelength for the chunk, and the minumum upper wavelength, over all observations
       λmin = maximum(map(obsid->first(clt.chunk_list[obsid].data[chid].λ), 1:length(clt) ))
       λmax = minimum(map(obsid->last( clt.chunk_list[obsid].data[chid].λ), 1:length(clt) ))
+      # extend λmin/λmax by the velocity range over which we don't want the mask to change
+      λmin  = λmin*calc_doppler_factor(plan.v_center + plan.v_range_no_mask_change)
+      λmax  = λmax*calc_doppler_factor(plan.v_center - plan.v_range_no_mask_change)
```

Comment:
Since these could be km/s rather than m/s, I think we should use calc_doppler_factor(plan.v_center)*calc_doppler_factor(plan.v_range_no_mask_change), just to avoid having to worry about this approximation in the future.